led_strip/README.md

```diff
@@ -27,13 +27,9 @@ led_strip_config_t strip_config = {
 };
 
 led_strip_rmt_config_t rmt_config = {
-#if ESP_IDF_VERSION < ESP_IDF_VERSION_VAL(5, 0, 0)
```

Comment:
Does this change mean we are already removing v4.x support? In that case we should update idf_component.yml.

Reply:
Not yet, I plan to remove the 4.x support in led_strip version 3 (still in feature collecting stage). I only removed it from the programming guide for now, as "not recommended".

Reply:
Okay, then I don't understand this part of the diff. Previously, for IDF versions <5.0, the initializer was
`led_strip_rmt_config_t rmt_config = {.rmt_channel = 0}`. Now, it will be `led_strip_rmt_config_t rmt_config = { .clk_src = RMT_CLK_SRC_DEFAULT, ... }`. Will this work in v4.4?